packages/flutter/lib/src/material/ink_splash.dart

```diff
@@ -46,10 +46,14 @@ double _getSplashRadiusForPositionInSize(Size bounds, Offset position) {
 ///
 /// This object is rarely created directly. Instead of creating an ink splash
 /// directly, consider using an [InkResponse] or [InkWell] widget, which uses
-/// gestures (such as tap and long-press) to trigger ink splashes.
+/// gestures (such as tap and long-press) to trigger ink splashes. This class
+/// is used when the [Theme]'s [ThemeData.splashType] is [InkSplashType.drop]
+/// (which is the theme's default splash type).
```

Comment:
"the theme's default splash type" is unclear; i'd probably just say "which is the default splash type" or some such.